jgrapht-io/src/main/java/org/jgrapht/io/ImportException.java

```diff
@@ -22,7 +22,7 @@
  */
 public class ImportException
```

Comment:
What is the reason for changing this to an unchecked exception?

Reply:
Checked exceptions become rather difficult to handle when combined with lambdas. Assuming proper documentation, I prefer to have them as runtime to avoid adding custom wrappers for checked lambdas.

Reply:
Got it.